web/gridt/models/user.py

```diff
@@ -32,6 +33,7 @@ class User(db.Model):
     username = db.Column(db.String(32))
     email = db.Column(db.String(40), unique=True, nullable=False)
     password_hash = db.Column(db.String(128))
+    email_hash = db.Column(db.String(128))
```

Comment:
Adding an email hash to the database might not be in our best interest, as it adds no new information to the database. This hash can always be derived from the email so storing them together only increases our database without expanding our knowledge. Theoretically it could be argued that if space is cheaper than computational power it could save resources, but that is an optimization that I would recommend against. 

This could also create a possible issue when the email would be changed. In that case the hash should also change, which right now would not happen.